LINUX/scripts/buildRpm.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+set -e
+PACKAGE=netmap
+
+if [[ $# -gt 3 || $# -lt 2 ]] ; then
+    echo 'Usage:  sh buildRpm <VERSION> <BUILD_NUMBER> <NETMAP-GIT-USER<optional>>'
+    exit 0
+fi
+ 
+VERSION="$1"
+BUILD="$2"
+
+if [ $# -eq 3 ]; then
+   USER=$3
+else
+   USER="LogRhythm"
+fi
+echo "USER IS $USER";
+
+echo "Building netmap for  USER: $USER, NETMAP VERSION: $VERSION"
```

Comment:
Not for this pull request, but for later when you want this to be merged back to master. 

We should have a dialogue with the netmap develpers for good git practices.  It seems that the current process on their side could improve by not always working on master.